### PR TITLE
Fix: Issues addressed in doc - Aug 20 - Aug 23

### PR DESCRIPTION
**Pull request by MiteshDarda**
## Description

Fixed Issues mentioned in the loop dodument Aug-20 to Aug-23

Issue Link : [Loop Link](https://loop.cloud.microsoft/p/eyJ3Ijp7InUiOiJodHRwczovL3RlY2hkb21lLnNoYXJlcG9pbnQuY29tLz9uYXY9Y3owbE1rWW1aRDFpSVd0V1RsRnRSM0I0U0RCeGQwZEdXa3BLZG05WldVeDBNMDlzYzNkMlkwcEVkVVZFYVRWa1gzQmFPWHB3YzFkSGNHMVFiazFTYjI4dGEyUXllV1ZZTTJjbVpqMHdNVXBVUkVvM1ZWZzNWRmhCVDFCUlNVZEpSa1ZMVmtwUU5EVlZWVU5PUmxORUptTTlKbVpzZFdsa1BURSUzRCIsInIiOmZhbHNlfSwicCI6eyJ1IjoiaHR0cHM6Ly90ZWNoZG9tZS5zaGFyZXBvaW50LmNvbS86Zmw6L3IvY29udGVudHN0b3JhZ2UvQ1NQXzk4NTA1MzkxLTcxNmEtNGExZi1iMDE4LTU2NDkyNmZhMTg2MC9Eb2N1bWVudCUyMExpYnJhcnkvTG9vcEFwcERhdGEvVW50aXRsZWQlMjAyLmxvb3A%2FZD13ZjI5ODYzNzBlZjc3NDAwOWE4YmE2MTBiNjM1MWEwZjMmY3NmPTEmd2ViPTEmbmF2PWN6MGxNa1pqYjI1MFpXNTBjM1J2Y21GblpTVXlSa05UVUY4NU9EVXdOVE01TVMwM01UWmhMVFJoTVdZdFlqQXhPQzAxTmpRNU1qWm1ZVEU0TmpBbVpEMWlJV3RXVGxGdFIzQjRTREJ4ZDBkR1drcEtkbTlaV1V4ME0wOXNjM2QyWTBwRWRVVkVhVFZrWDNCYU9YcHdjMWRIY0cxUWJrMVNiMjh0YTJReWVXVllNMmNtWmowd01VcFVSRW8zVlZSUlRVOU5VRVUxTjFCQ1JrRkxVazlVUWtKT1VsWkVTVWhVSm1NOUpUSkdKbVpzZFdsa1BURW1ZVDFNYjI5d1FYQndKbkE5SlRRd1pteDFhV1I0SlRKR2JHOXZjQzF3WVdkbExXTnZiblJoYVc1bGNpWjRQU1UzUWlVeU1uY2xNaklsTTBFbE1qSlVNRkpVVlVoNE1GcFhUbTlhUnpsMFdsTTFlbUZIUm5sYVdFSjJZVmMxTUV4dFRuWmlXSGhwU1ZkMFYxUnNSblJTTTBJMFUwUkNlR1F3WkVkWGEzQkxaRzA1V2xkVmVEQk5NRGx6WXpOa01sa3djRVZrVlZaRllWUldhMWd6UW1GUFdIQjNZekZrU0dOSE1WRmlhekZUWWpJNGRHRXlVWGxsVjFaWlRUSmtPRTFFUmt0V1JWSkxUakZXV1U0eFVsbFJWVGxSVlZWc1NGTlZXa1pUTVZwTFZVUlJNVlpXVmtSVWExcFVVa0VsTTBRbE0wUWxNaklsTWtNbE1qSnBKVEl5SlROQkpUSXlZVFkxT1dZeFpUTXRNbVpoWVMwMFpERmtMV0ZpTUdZdE56TTBaR0ZsTjJRM09HWmpKVEl5SlRkRSIsInIiOmZhbHNlfSwiaSI6eyJpIjoiYTY1OWYxZTMtMmZhYS00ZDFkLWFiMGYtNzM0ZGFlN2Q3OGZjIn19)

## What type of PR is this? (check all applicable)

- [ ] 🍕 Feature
- [ ] 🐛 Bug Fix
- [ ] 📝 Documentation Update
- [ ] 🎨 Style
- [ ] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [ ] ✅ Test
- [ ] 🤖 Build

## Have you tested locally?

- [ ] 👍 yes
- [ ] 🙅 no, because I am lazy

## Added tests?

- [ ] 👍 yes
- [ ] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help

## Added to documentation?

- [ ] 📜 README.md

### CODE REVIEW

**Comment by pratikshinde590** [comment]
Please attach the issue link

**Comment by pratikshinde590** [comment]
Please attach the issue link

**Comment by pratikshinde590** [comment]
Please attach the issue link

**Comment by pratikshinde590** [comment]
Please attach the issue link

**Comment by pratikshinde590** [comment]
Please attach the issue link

**Comment by pratikshinde590** [comment]
Please attach the issue link

**Review by gc-codes — APPROVED** [review]

File: src/packages/@app/pages/Auth/entry-point/EntryPoint.svelte [hunk 1]
```diff
@@ -84,17 +86,33 @@
...
-        >Sparrow website <img class="mx-2" src={externalLink} alt="" /></span
-      >
+        class="text-primary-300 cursor-pointer text-decoration-underline"
+        >Sparrow website
+        <span class="mx-2">
+          <ExternalLinkIcon
+            width="16px"
+            height="16px"
+            color="var(--text-primary-300)"
+          />
```
Comment (itsmdasifraza) [inline src/packages/@app/pages/Auth/entry-point/EntryPoint.svelte:96]: use --icon here

File: src/packages/@app/pages/Auth/entry-point/EntryPoint.svelte [hunk 2]
```diff
@@ -84,17 +86,33 @@
...
           notifications.success("Link copied to clipboard!");
-        }}><img src={copyIcon} class="px-2" />Copy</span
+        }}
+      >
+        <span class="mx-2">
+          <CopyIcon
+            width="18px"
+            height="18px"
+            color="var(--text-primary-300)"
+          />
```
Comment (itsmdasifraza) [inline src/packages/@app/pages/Auth/entry-point/EntryPoint.svelte:113]: --icon

File: src/packages/@library/icons/ExternalLink.svelte [hunk 3]
```diff
@@ -0,0 +1,18 @@
+<script>
+  export let height = "18px";
+  export let width = "18px";
+  export let color = "grey";
```
Comment (itsmdasifraza) [inline src/packages/@library/icons/ExternalLink.svelte:4]: make it to default 24px 24px grey, using this in all the icons to maintain consistency

**Comment by pratikshinde590** [comment]
Please attach the issue link

**Review by itsmdasifraza — APPROVED** [review]

**Review by Astitva877 — APPROVED** [review]

**Pull request merged**